README.md

```diff
@@ -386,6 +386,33 @@ The most relevant ones are:
 
 * `--help`: displays a help message listing all the available options.
 
+* `--plot_ascii_graph: plots the latencies in a ASCII art type of flow, like
+  ```
+  10:17:35 34:13 (2053) > +0839 ......... |
```

Comment:
There's a space between the last dot and the bar (|). Shouldn't this be a dot too? Else, what does a space mean?

Reply:
Done.